modules/caddyhttp/fileserver/caddyfile.go

```diff
@@ -37,7 +35,7 @@ func init() {
 // server and configures it with this syntax:
 //
 //	file_server [<matcher>] [browse] {
-//	    fs            <backend...>
+//	    fs            <filesystem>
```

Comment:
I think technically this is a breaking change, right? Because we now need to define an fs name instead of an fs module? I think this is ok because it's an improvement, we just need to document it in the release notes.

Reply:
yes it is. I think this makes a lot more sense.

I tried playing around to declare filesystems inline but it was a nightmare so I gave up...